src/main/java/frc/robot/commands/AutoIndexerShootCommand.java

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants.IndexerConstants;
+import frc.robot.Constants.ShooterConstants;
+import frc.robot.subsystems.IndexerSubsystem;
+import frc.robot.subsystems.ShooterSubsystem;
+
+public class AutoIndexerShootCommand extends Command {
+    private ShooterSubsystem shooter;
+    private IndexerSubsystem indexer;
+    private Timer timer;
+    private Timer hasNoteTimer;
+    private Boolean hadNote;
+
+    /** Creates a new AutoShoot. */
+    public AutoIndexerShootCommand(ShooterSubsystem shooter, IndexerSubsystem indexer) {
+        this.shooter = shooter;
+        this.indexer = indexer;
+        this.timer = new Timer();
+        this.hasNoteTimer = new Timer();
+        this.hadNote = hadNote;
+        addRequirements(shooter, indexer);
+    }
+
+    // Called when the command is initially scheduled.
+    @Override
+    public void initialize() {
+        timer.stop();
+        timer.reset();
+        hasNoteTimer.stop();
+        hasNoteTimer.reset();
+        hadNote = false;
+    }
+
+    // Called every time the scheduler runs while the command is scheduled.
+    @Override
+    public void execute() {
+        hasNoteTimer.start();
```

Comment:
This timer doesn't seem to do anything. But I suppose it's harmless.

Reply:
oh yes, missed this. I will remove